index.html

```diff
@@ -36,6 +36,71 @@
       </nav>
     </div>
   </header>
+  <div class = "section1"> 
```

Comment:
What do you think of having a class name that better reflects this section? Such a name will be something that if you are going through just the styles in `.styles.css` you can easily assume which section of the app each styling refers.

Maybe something like `class="trending"` since this is the trending section 🤔

Reply:
Very true, naming seems easy
But it's one of the most difficult to be sincere,I'll start thinking about naming more thoroughly now.